specification/vmware/Microsoft.AVS/routes.tsp

```diff
@@ -0,0 +1,1362 @@
+import "@azure-tools/typespec-azure-core";
+import "@typespec/rest";
+import "./models.tsp";
+import "./parameters.tsp";
+
+using TypeSpec.Rest;
+using TypeSpec.Http;
+using Azure.ResourceManager;
+
+namespace Microsoft.AVS;
+
+interface Operations extends Azure.ResourceManager.Operations {}
+
+interface Locations {
+  @doc("Return trial status for subscription by region")
```

Comment:
You will likely want to use either the checkGlobal or checkLocal name availability operation templates: https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-resource-manager/lib/operations.tsp#L435-L471